metadata-ingestion/src/datahub/ingestion/source/mongodb.py

```diff
@@ -2,10 +2,6 @@
 from dataclasses import dataclass, field
 from typing import Dict, Iterable, List, Optional, Tuple, Type, Union, ValuesView
 
-import bson
-import bson.dbref
-import bson.int64
```

Comment:
Were these just unused?

Reply:
Ah I just ran the intellij "optimize imports" command on a bunch of files had imports to delete. It's very possible these are needed, we'll see what CI says I guess